ush/regrid_gsiSfcIncr_to_tile.sh

```diff
@@ -79,63 +73,196 @@ else # construct restart times for deterministic member
+        mkdir -p "${memdir}"
+
+        if [[ "${imem}" -gt 1 ]]; then
+            in_dir+=", "
+        fi
+        in_dir+="\"./${memchar}/\""        
+    else
+        # If deterministic job, memdir is just DATA
+        memdir="${DATA}"
+
+        in_dir="'./'"
     fi
 
+    # Create MPMD command file for this member
+    rm -f "cmdfile.${imem}"
+    touch "cmdfile.${imem}"
+    chmod 755 "cmdfile.${imem}"
+    echo "#!/bin/bash" > "cmdfile.${imem}"
+
     for FHR in "${soilinc_fhrs[@]}"; do
+        echo "cpreq ${COMIN_SOIL_ANALYSIS_MEM}/${APREFIX_ENS}increment.sfc.i00${FHR}.nc \
+                    ${memdir}/sfci00${FHR}.nc" >> "cmdfile.${imem}"
+    done 
+
+    if [[ "${DO_LAND_IAU}" = ".true." ]]; then 
+        for FHI in "${landifhrs[@]}"; do
+            echo "cpreq ${COMIN_SOIL_ANALYSIS_MEM}/${APREFIX_ENS}increment.sfc.i00${FHI}.nc \
+                        ${memdir}/sfci00${FHI}.nc" >> "cmdfile.${imem}"
+        done
+    fi
+
+    # Append this member's command file to master command file
+    echo "${DATA}/cmdfile.${imem}" >> cmdfile
+done
+
+# Run MPMD to stage input files
+"${USHgfs}/run_mpmd.sh" "cmdfile" && true
+export err=$?
+if [[ ${err} -ne 0 ]]; then
+    err_exit "run_mpmd.sh failed!"
+fi
+
+# Finish defining input/output directory list
+export out_dir="${in_dir}"
 
+#
+# Regrid soil increments and save to COMOUT
+#
+
+# Increments for offline analysis
+# If land IAU --> deterministic only. If no land IAU --> both deterministic and ensemble
+if [[ "${DO_LAND_IAU}" = ".false." || "${RUN}" == "gdas" || "${RUN}" == "gfs" ]]; then
+    for FHR in "${soilinc_fhrs[@]}"; do
+        # Set namelist variables
         export add_time_dim=".false."
         export time_list="${FHR}"
+        export out_fname="'sfci00${FHR}'"
 
+        # Create regrid namelist
         rm -f "regrid.nml"
         atparse < "${regrid_nml_tmpl}" >> "regrid.nml"
 
-        cpreq "${COMIN_SOIL_ANALYSIS_MEM}/${APREFIX_ENS}increment.sfc.i00${FHR}.nc" \
-               "${DATA}/enkfgdas.sfci00${FHR}.nc"
-
+        # Run regrid executable
         ${APRUN_REGRID} "${REGRID_EXEC}" "${REDOUT}${PGMOUT}" "${REDERR}${PGMERR}"
+    	export err=$?
+	    if [[ ${err} -ne 0 ]]; then
+	        err_exit "${REGRID_EXEC} failed, ABORT!"
+	    fi
+    done
+fi
 
-        for n in $(seq 1 "${ntiles}"); do
-            cpfs "${DATA}/sfci.tile${n}.nc"  "${COMOUT_ATMOS_ANALYSIS_MEM}/increment.sfc.i00${FHR}.tile${n}.nc"
-        done
-    done 
+# Increments for forecast job with land IAU
+# If land IAU --> deterministic and ensemble
+if [[ "${DO_LAND_IAU}" = ".true." ]]; then
+    # Set namelist variables
+    export add_time_dim=".true."
+    export time_list="${IAUFHRS}"
+    export out_fname="'sfci'"
+
+    # Create regrid namelist
+    rm -f "regrid.nml"
+    atparse < "${regrid_nml_tmpl}" >> "regrid.nml"
+
+    # Run regrid executable
+    export pgm="${REGRID_EXEC}"
+	${APRUN_REGRID} "${REGRID_EXEC}" "${REDOUT}${PGMOUT}" "${REDERR}${PGMERR}"
```

Comment:
`REDOUT` and `REDERR` need to be removed from the workflow.  They don't work as intended.

For example, at a terminal:
```console
> REDOUT="1>"
> REDERR="2>" 
> PGMOUT=pgmout
> PGMERR=pgmerr
> set -x
> ./gw_setup.sh "${REDOUT}${PGMOUT}" "${REDERR}${PGMERR}"
+ ./gw_setup.sh '1>pgmout' '2>pgmerr'                                                                                                                                                     
Running "module reset". Resetting modules to system default. The following $MODULEPATH directories have been removed: None
> ls pgm*
+ ls pgm*
ls: cannot access 'pgm*': No such file or directory
```

Instead, just use `1>` and `2>` explicitly.  I'll open an issue to remove `REDOUT` and `REDERR`.

```suggestion
    ${APRUN_REGRID} "${REGRID_EXEC}" 1>"${PGMOUT}" 2>"${PGMERR}"
```

Reply:
Opened https://github.com/NOAA-EMC/global-workflow/issues/4230